### ISSUE TITLE
Disconnect Method

### ISSUE THREAD

**Issue by stossy11** [spotlightishere/SwordRPC#4]
I'm building an iOS app that talks to a desktop app. I need a way to disconnect from the RPC. Is there a way to add a disconnect method for this?

**Comment by spotlightishere** [comment]
Absolutely! Would the changes in f50152e9c32466c3f8b8295c51e21bb50e0cb961 work for your needs?

Unfortunately, much of this code is hacked together for use with [Ongaku](https://github.com/spotlightishere/Ongaku). If you find any feature that isn't implemented correctly or goes fully unimplemented, please feel free to file further issues - would love to hear!
